src/analytics/formTracker/formTracker.ts

```diff
@@ -6,7 +6,7 @@ export class FormTracker extends BaseTracker {
   DROPDOWN_FIELD_TYPE = "drop-down list";
   RADIO_FIELD_TYPE = "radio buttons";
 
-  constructor() {
+  constructor(isDataSensitive: boolean = false) {
```

Comment:
Is this still required as the value is set in formResponseTracker? If it is, should it be true?

Reply:
ah this was legacy from when we were thinking of extracting this out of the form response tracker, i've removed!